Resources/Textures/DeltaV/Clothing/Uniforms/Jumpskirt/cjformaldress.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC0-1.0",
+  "copyright": "Made by HTMLSystem, edited from https://github.com/ParadiseSS13/Paradise/",
```

Comment:
is this accurate? it sounds like it could read "made by paradisess13, edited by HTMLSystem" unless im major wrong

Reply:
true also paradise assets are CC-BY-SA-3.0 i think so this cant be CC0-1.0, also use a permalink to the dmi its based on

Reply:
I basically just copy pasted what the HoP formal suit says in it's copyright thing, but yeah I can edit it.

Reply:
I lied, appearently I just fucked up the copy/pasting somehow